helpers/auth.ts

```diff
@@ -0,0 +1,103 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jwt-simple');
+
+const saltRounds: number = 10;
+const JWT_SECRET: string = 'secret'; // TODO: Integrate .env
```

Comment:
I will fix this later in a separate PR for all global setup/config related issues